pages/partials/jobSequenceResults.ejs

```diff
@@ -23,24 +23,36 @@
     $(function() {
         var socket = io();
 
-        socket.on('update', function() {window.location.reload(true);});
-        socket.on('change:output', function(msg) {$("#output-" + msg.job_id).html(msg.output);});
+        socket.on('update', function() {
+            window.location.reload(true);
+        });
+        socket.on('change:output', function(msg) {
+            $('#output-' + msg.job_id).html(msg.output);
+        });
 
         // Join the rooms for the job_sequence and for each individual job.
         // Check return values for a change that happened since we rendered the page
         // and reload if anything changed. This avoids the race condition where
         // jobs change between page render and socket activation.
 
-        socket.emit('joinJobSequence', {job_sequence_id: <%= job_sequence.id %>}, function(msg) {
-            if (msg.job_count != '<%= job_sequence.job_count %>') {window.location.reload(true);}
+        socket.emit('joinJobSequence', {
+            job_sequence_id: '<%= job_sequence.id %>',
+            token: '<%= job_sequence.token %>',
```

Comment:
Most changes here were just reformatting; the addition of the `token` properties here and below should be the only functional changes.